dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -853,6 +853,30 @@ static boolean doesOrganizationExistToUser(Long organizationId, Long userId, Org
     }
 
 
+    static boolean isUserAdminOrMaintainer(Organization organization, Long userId) {
+        if (organization == null) {
+            String msg = "Organization not found";
+            LOG.info(msg);
+            throw new CustomWebApplicationException(msg, HttpStatus.SC_NOT_FOUND);
+        }
+
+        OrganizationUser organizationUser = getUserOrgRole(organization, userId);
+        if (organizationUser == null) {
+            return false;
+        }
+
+        if (organizationUser.getRole() == OrganizationUser.Role.ADMIN || organizationUser.getRole() == OrganizationUser.Role.MAINTAINER) {
+            return true;
+        }
+        return false;
+    }
+
+    static boolean isUserAdminOrMaintainer(Long organizationId, Long userId, OrganizationDAO organizationDAO) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6348842)